src/kata3.numberToReversedDigits.js

```diff
@@ -1,6 +1,7 @@
 const numberToReversedDigits = numbers => {
 	let wordNums = String(numbers);
 	let array = [];
+	// review
```

Comment:
Use a more helpful variable name than `i` here for readability. You might also want to consider these methods:

[String.prototype.split()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)
[Array.prototype.reverse()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse)